tests/translator/output/error_http_api_with_invalid_auth_while_method_auth_is_none.json

```diff
@@ -1,3 +1,3 @@
 {
-  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [MyApi] is invalid. Property 'Auth' should be a map."
+  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 3. Resource with id [MyApi] is invalid. Property 'Auth' should be a map. Resource with id [MyRestApi] is invalid. Property 'Auth' should be a map. Resource with id [MyRestApi] is invalid. Type of property 'Auth' is invalid."
```

Comment:
We have duplicate error messages here, and it is caused by two `sam_expect` happening in two different resources. I will create a follow up to deduplicate the error messages if there is no better idea.

Reply:
https://github.com/aws/serverless-application-model/pull/2748